package_test.go

```diff
@@ -766,6 +766,13 @@ func (s *PackageSuite) TestGetAllErrors(c *C) {
 		inputs:  []any{},
 		slices:  []any{&[]Person{}},
 		err:     `cannot populate slice: output variables provided but not referenced in query`,
+	}, {
+		summary: "nothing returned",
+		query:   "SELECT &Person.* FROM person WHERE id = $Person.id",
+		types:   []any{Person{}},
+		inputs:  []any{Person{ID: 1243321}},
+		slices:  []any{&[]*Person{}},
+		err:     "cannot populate slice: sql: no rows in result set",
```

Comment:
Sorry I missed this on a previous review but should we assert that the error is `ErrNoRows`. If we have the same message but not the exact error we could break the public API. The rest is perfect!

Reply:
Changed.